pkg/operator/controller.go

```diff
@@ -477,6 +476,14 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 		c.kubeconfig, c.clients, c.listers,
 	)
 
+	imageRegistryCertificatesController := NewImageRegistryCertificatesController(
+		c.clients.Core,
+		kubeInformerFactory.Core().V1().ConfigMaps(),
+		kubeInformerFactory.Core().V1().Services(),
```

Comment:
I try to use the upstream approach for controllers. I think their code is cleaner and we have a lot to learn from them.

https://github.com/kubernetes/kubernetes/blob/3758426884e3c82cbd99c72e8015f4396f21fde2/cmd/kube-controller-manager/app/apps.go#L40

In this case `v1.Interface` instead of two informers will be much more opaque dependency. I don't want to read this code few month later and think:

Do it really wants to use all corev1 types?! I don't think so, but what kind of objects does this controller really need? Meh, I need to read its code...